webpack/assets/javascripts/compute_resource/ovirt.js

```diff
@@ -68,6 +68,17 @@ export function instanceTypeSelected(item) {
         $('[id$=_ha]').prop('checked', result.ha);
         const instanceTypeSelector = $('#host_compute_attributes_instance_type');
 
+        if (result.name != null) {
+          $('[id$=_memory]').attr('disabled', 'disabled');
+          $('[id$=_cores]').attr('disabled', 'disabled');
+          $('[id$=_sockets]').attr('disabled', 'disabled');
+          $('[id$=_ha]').attr('disabled', 'disabled');
+        } else {
+          $('[id$=_memory]').removeAttr('disabled');
+          $('[id$=_cores]').removeAttr('disabled');
+          $('[id$=_sockets]').removeAttr('disabled');
+          $('[id$=_ha]').removeAttr('disabled');
+        }
```

Comment:
Please use `.prop('disabled', <true or false>)` instead of `attr` and `removeAttr`. Building on @lizagilman's suggestion, you can do it all in one:
```js
[ _'memory', '_cores', '_sockets', '_ha'].forEach((name)=> 
  $(`[id$=${name}]`).prop('disabled', (result.name != null)))